about.md

```diff
@@ -1,37 +1,12 @@
 ---
 layout: about
 title: About
-description: Learn all about the Markdown Guide.
+description: Learn about the origins of TruCol and the team behind TruCol.
 ---
 
-## Purpose
+## The origins of TruCol
 
-*The Markdown Guide* is a comprehensive Markdown reference designed for both novices and experts. It was born out of frustration with existing Markdown references that are incomplete, inadequate, or both.
+TruCol was founded as a project for the ETHDenver Hackathon as a collaboration between four students interested in the development of Ethereum projects. After competing in various categories, and successfully claiming multiple bounties, the team decided to continue with the development of the project. Other students who were interested in the project joined the team and several blockchain-related organisations were interested in the project.
```

Comment:
We experienced that we did not always have the time or a skill for specific coding work ourselves when setting up our own company. Setting out bounties would provide a great solution for this. However, existing systems require an often costly middleperson and a modicum of trust between all parties. This is why we set out to develop a decentralized, trust-less collaboration platform and protocol for test-driven programming development for the ETHDenver Hackathon as a collaboration between four students interested in the development of Ethereum projects. After competing in various categories, and successfully claiming multiple bounties, the team decided to continue with the development of the project.

Reply:
I like your suggestion, except for one sentence ( see bold sentence below), I think it is too long.
```
We experienced that we did not always have the time or a skill for specific coding work ourselves when setting up our own company. Setting out bounties would provide a great solution for this. However, existing systems require an often costly middleperson and a modicum of trust between all parties. ** This is why we set out to develop a decentralized, trust-less collaboration platform and protocol for test-driven programming development for the ETHDenver Hackathon as a collaboration between four students interested in the development of Ethereum projects **. After competing in various categories, and successfully claiming multiple bounties, the team decided to continue with the development of the project.
```
I would like to suggest this change, let me know what you think

```
We experienced that we did not always have the time or a skill for specific coding work ourselves when setting up our own company. Setting out bounties would provide a great solution for this. However, existing systems require an often costly middleperson and a modicum of trust between all parties. **This is why we set out to develop a decentralized, trust-less collaboration platform and protocol for test-driven programming development. The project started at the ETHDenver Hackathon as a collaboration between four students interested in the development of Ethereum projects.** After competing in various categories, and successfully claiming multiple bounties, the team decided to continue with the development of the project.
```

Reply:
Excellent!